addons/mail/static/src/core/common/chat_window.js

```diff
@@ -132,6 +132,12 @@ export class ChatWindow extends Component {
                 }
                 break;
             }
+            case "k":
+                if (ev.ctrlKey) {
+                    this.store.env.services.command.openMainPalette({searchValue: "@"});
```

Comment:
```suggestion
                    this.store.env.services.command.openMainPalette({ searchValue: "@" });
```
You might want to install the JS tooling (`web/tooling/enable.sh`).